tests/test_customized/TestControllers.py

```diff
@@ -178,3 +178,19 @@ def test_customize_test_creates_with_customize_regression_tests(self, mock_user,
             regression_tests = test.get_customized_regressiontests()
             self.assertIn(2, regression_tests)
             self.assertNotIn(1, regression_tests)
+
+    def test_customize_test_wrong_commit_hash(self, mock_user, mock_git, mock_requests):
+        import mod_customized.controllers
+        reload(mod_customized.controllers)
+        self.create_user_with_role(
+            self.user.name, self.user.email, self.user.password, Role.tester)
+        with self.app.test_client() as c:
+            response = c.post(
+                '/account/login', data=self.create_login_form_data(self.user.email, self.user.password))
+            response = c.post(
+                '/custom/', data=self.create_customize_form('hellofromtheotherside', ['linux'],
+                                                            regression_test=[2]), follow_redirects=True)
+            self.assertEqual(response.status_code, 200)
+            test = Test.query.filter(Test.id == 3).first()
+            self.assertEqual(test,None)
```

Comment:
Why is this needed here?

Reply:
To Test that it doesn't create create another customized test.

Reply:
Sorry, how does this test that?

Reply:
There are 2 tests added by default in base.py (line 114). If a test would be added, a test of id 3 should be created which means test = Test.query.filter(Test.id == 3).first() would not return None

Reply:
1. In that case shouldn't it be `CustomizedTest` instead of `Test`?
2. What happens if in future we add one or more default tests? Isn't it better to search for the commit hash?
3. Also, if we're just checking if commit hash is being verified fine, I don't think testing that even matters.

Reply:
@saurabhshri so should I remove that part?